.gitignore

```diff
@@ -1,3 +1,5 @@
 __pycache__
 subprojects
 bin*
+*~
+build
```

Comment:
Need to add `*.pyc` as well. We don't want our repository keeping those as they are automatically generated.